docs/advanced-options.md

```diff
@@ -4,7 +4,7 @@
 [Video Options](#video-options)  
 [Dynamic Discovery](#dynamic-discovery)  
 [Camera Paths](#keep-the-paths-of-existing-camera-up-to-date) 
```

Comment:
2 spaces need to be added after this line for `RTSP Server` to move down a line.